frontend/src/layouts/BaseLayout.vue

```diff
@@ -192,6 +192,28 @@
                 </base-tooltip>
               </span>
             </p>
+
+            <!-- Send history toggle -->
+            <p>
+              <q-toggle
+                @update:modelValue="toggleSendHistory"
+                :modelValue="sendHistorySave"
```

Comment:
It looks like it's supposed to default to true (enabled) but from the UI it seems to default to disabled

Reply:
https://github.com/ScopeLift/umbra-protocol/pull/560/commits/9cdd87b565b1bb8142e82ad76ed62705054cb6f3 ah good catch